dist/stat_data.py

```diff
@@ -680,6 +680,7 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     TxnStat('txn_begin', 'transaction begins'),
     TxnStat('txn_commit', 'transactions committed'),
+    TxnStat('txn_ckpt_tree_duration', 'transaction checkpoint tree duration (usecs)'),
```

Comment:
Thanks for fixing the description string. It still needs to be moved to the `CheckpointStat` section though.